doc/instore-app/available-settings/settings.md

```diff
@@ -0,0 +1,166 @@
+---
+slug: /poscreators/middleware-doc/instore-app/available-settings
+title: 'Available settings'
+---
+
+## InStore App – Available Settings (v1.2.6)
+
+:::info Important
+
+All settings and options described here apply **only** to version **1.2.6** of the InStoreApp.  
+If you are using an older version, some settings may differ or may not be available at all.
+
+:::
+
+### DeviceId
+Shows the unique identifier of this device. Quote this ID when contacting support or managing a fleet. *(Read‑only.)*
+
+### DeviceType
+Displays the device model/type (e.g., terminal vendor and model). Useful to identify hardware-specific behavior. *(Read‑only.)*
+
+### IP Address
+Shows the current IP address of the device on your network. Use it when setting up network printing or firewall rules. *(Read‑only.)*
+
+---
+
+## Base settings
+
+### CashBoxId
+The CashBox the app is paired with. It links the app to the correct fiscal/payment configuration.
+
+### Unpair CashBox
+Disconnects the current CashBox to pair another one (e.g., when moving the device to a new store).
+
+### Refresh DeviceConfiguration
+This button updates your device with the current backend configuration without requiring an app restart.
+
+### End App
+Closes the app gracefully and persists configuration changes.  
+*Note:* Closing during an active process (e.g., printing) may interrupt that process, finish ongoing tasks first.
+
+### Terminal ID Filter
+Filters receipts and data for a specific terminal within the CashBox.  
+For advanced multi-terminal options, see [Multiterminal Settings](https://docs.fiskaltrust.eu/docs/poscreators/middleware-doc/instore-app/multiterminal-settings)
+
+### Operation Mode
+Defines which features are available in the UI. Some options and sections appear or hide based on this selection:
+
+- **Consumer** – Customer-facing (receipt display/printing). Allows changing **Print Delay**, which is not available in Merchant mode.
+- **Merchant** – Merchant/admin. Cannot change Print Delay, but offers extra gestures:  
+  - Pull **down** on the QR code screen to show the receipt.  
+  - Pull **up** to reveal action buttons.
+
+
+### Use local configuration
+
+If enabled, local device settings override the backend configuration.
+
+If disabled, the app follows centrally managed (backend) settings.
+
+:::info Note
+
+Currently, this setting can only be configured by internal staff via the backend. Customers do not yet have the ability to change this setting themselves.
```

Comment:
I think this is wrong. Customers can currently not modify the backend based configuration themselves but they of course can configure the "Use local configuration" switch.
```suggestion
Currently, only internal fiskaltrust staff is able to modify the backend based configuration. Customers do not yet have the ability to configure those. Therefore this switch should stay disabled unless told by fiskaltrust otherwise.
```